packages/frontend/core/src/modules/peek-view/view/doc-preview/doc-peek-view.tsx

```diff
@@ -156,6 +164,7 @@ function DocPeekPreviewEditor({
               className={styles.editor}
               mode={mode}
               page={doc.blockSuiteDoc}
+              readonly={readonly}
```

Comment:
I have a question here, why not marking `doc.blockSuiteDoc` as `readonly`, but adding a new prop in `BlockSuiteEditor`?

Reply:
readonly should be editor scope state

Reply:
1. readonly should be a state of the editor

2. readonly should be a calculated value, not a variable that can be set.

some places write `doc.readonly = true` or `doc.readonly = false`, which make the code difficult to maintain.
eg. it might make the document in `trash` be editable

Reply:
I see. I guess there is no reason of using `doc.readonly` any longer?